src/tests/end-to-end/common/page.ts

```diff
@@ -8,9 +8,21 @@ import { DEFAULT_NEW_PAGE_WAIT_TIMEOUT_MS, DEFAULT_PAGE_ELEMENT_WAIT_TIMEOUT_MS
 
 export class Page {
     constructor(private readonly underlyingPage: Puppeteer.Page) {
+        underlyingPage.on('error', error => {
+            forceTestFailure(`error occurred - ${error.message}`);
```

Comment:
forceTestFailure doesnt work. I will send another PR to use jest done.fail call instead

Reply:
This PR is small.. why not have `done.fail` change in here too?